text/0012-cva.md

```diff
@@ -0,0 +1,161 @@
+
+### 4. Establish a certification program
+
+A validator certification program serves to impart the values underpinning the network's identity whilst detailing the duties & responsibilities expected of a validator during the course of their active engagement with the network.  It will revolve around a training program combined with a series of community inductions.
+
+The operational component of the training program will focus upon the mechanics of securely and safely operating a validator node.  The reputational component will focus upon participation within the wider social consensus process.  The technical component will focus upon imparting information regarding node software design & behavioural analysis.  The governance component will focus upon detailing the range of potential scenarios in which governance mechanisms impact upon operational concerns and how to respond to those scenarios in a timely manner.  
+
+The certification program would be available online.  The content might be a mix of materials augmented by a set of relatively simple tests to ensure that the materials were at least reviewed.  Successful completion of the certification program would be recorded and placed upon a public record, perhaps via the VDAO outlined above.  
+
+## Reference-level explanation
+
+[reference-level-explanation]: #reference-level-explanation
+
+Establishment of a Casper Validator's Association plus associated legal, technical and communications infrastructure is considered primarily a soft design problem.  It assumes that the proposal would be accepted by prospective Genesis set validators - by no means a given.  The design solution necessarily combines a diverse skillset: 
+
+- legal expertise;
+- community building;
+- smart contract development (see VDAO);
+- long-term thinking;
+
+Successful implementation will require clear signals from the CasperLabs management team being transmitted to the core set of validators assisting in the incubation of the network.  If stakeholders can relatively quickly arrive at consensus on both the broad and finer strokes of a proposal, then the association  may be established in a timely manner, i.e. before main-net.  It may even be possible to establish an embryonic VDAO before main-net but more likely this would be released at a later date. 
+ 
+## Drawbacks
+
+[drawbacks]: #drawbacks
+
+1. The proposals outlined in this CEP are potentially problematic from a political & jurisdictional perspective.  The crypto space is unique in that attracts a fair number of non-regulated entities, such entities may very likely be put off by the idea of mandatory association membership.
+
+2. The notion that validators may be classified in some respect, e.g. regulated | certified | other, requires some form of arbiter.  This is problematic if, for example, the arbiter is found to be byzantine.  
```

Comment:
Could use a significant expansion of this and quite a few more bullet points in the section.